cmake/qtifw/install_unix_createsymlinks.qs

```diff
@@ -68,7 +68,9 @@ function Component()
 
       // man page:
       linktarget = "/usr/local/share/man/man1";
-      component.addElevatedOperation("Execute", "ln", "-sf", "@TargetDir@/energyplus.1", linktarget,
+      component.addElevatedOperation("Execute", "mkdir", "-p", linktarget);
+
+      component.addElevatedOperation("Execute", "ln", "-sf", "@TargetDir@/energyplus.1", linktarget + "/energyplus.1",
```

Comment:
Install man page on mac (and linux): create the /usr/local/share/man/man1 directory if it doesn't exist (it doesn't by default) to you avoid this annoying warning

```
[24504] Operation \"Execute\" with arguments \"ln; -sf; @TargetDir@/energyplus.1; /usr/local/share/man/man1; UNDOEXECUTE; rm; /usr/local/share/man/man1/energyplus.1\" failed: Execution failed (Unexpected exit code: 1): \"ln -sf /Applications/EnergyPlus-22-2-0/energyplus.1 /usr/local/share/man/man1\"
[24504] installationErrorWithCancel : Installer Error : Error during installation process (Symlinks):
Execution failed (Unexpected exit code: 1): "ln -sf /Applications/EnergyPlus-22-2-0/energyplus.1 /usr/local/share/man/man1" Retry|Ignore|Cancel
```